shared/trainer/defaults.yaml

```diff
@@ -214,8 +219,8 @@ train:
       ent_var: 0.1 # the proportion of variance of entropy map in the entropy measure for a certain picture
       adv_main: 1.0 # the coefficient of the AdvEnt loss that minimize the entropy of the image by adversarial training
       adv_aux: 0.0 # the corresponding coefficient of the AdvEnt loss of second output
-      dis_main: 0.5 # the discriminator take care of the first output in the adversarial training
-      dis_aux: 0.5 # the discriminator take care of the second output in the adversarial training
+      dis_main: 1 # the discriminator take care of the first output in the adversarial training
```

Comment:
We're not using dis_main and dis_aux in the code right now right? And maybe we should get rid of ent_aux and adv_aux also since we're not using them?

Reply:
Good point, if we're not using parameters we might as well clean them away